src/kristin_tukun_ex/ex_04/myrand.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Kristin Tukun'
+__email__ = 'kristin.tukun@nmbu.no'
+
+
+class LCGRand:
+
+    def __init__(self, number_seed):
+
+        self.random_number = number_seed
+
+    def rand(self):
+        a = 7 ** 5
+        m = 2 ** 31 - 1
```

Comment:
Instead of recalculating these numbers everytime `rand()` is called, you can put it before the `__init__` function (but still within this class) and reuse it everytime `rand()` is called